### PR TITLE
Adds stage-1 compatibility

### PR DESCRIPTION
**Pull request by abhi-govula**
Adds ES6 stage-1 compatibility

### CODE REVIEW

File: lib/PropTransformer.js [hunk 1]
```diff
@@ -21,11 +21,15 @@ export default class PropTransformer extends Component {
     );
 
     if (Array.isArray(this.props.children)) {
-      return transformedChildren;
+      return <div>{transformedChildren}</div>;
```
Comment (sagiavinash) [inline lib/PropTransformer.js]: no need of this, react 16 supports returning arrays and null

Comment (sagiavinash) [reply lib/PropTransformer.js]: i added react@16.0.0 as peer dependency

Comment (abhi-govula) [reply lib/PropTransformer.js]: yes, but often times we ignore the install peer dependancy error (like i did) and this will just make it fail safe

Comment (sagiavinash) [reply lib/PropTransformer.js]: @abhi-govula can u make it a `span`, block element affects layout if child was an inline element

File: lib/PropTransformer.js [hunk 2]
```diff
@@ -21,11 +21,15 @@ export default class PropTransformer extends Component {
...
     } else if (this.props.children) {
       return transformedChildren[0];
     }
 
+
     return null;
```
Comment (sagiavinash) [inline lib/PropTransformer.js]: @abhi-govula can u make it `span` instead of null

**Pull request merged**